components/Page.vue

```diff
@@ -25,6 +25,8 @@
                 :size="sizes[0]"
                 :tab-size="editor.tabSize"
                 :language="editor.language"
+                :filename="editor.filename"
+                :allow-filename="editors.length > 1"
```

Comment:
Instead of a separate prop for `allow-filename`, can we use a ternary condition to simply pass in `null` in the `filename` prop? For example:

```vue
<Editor
  :filename="editors.length > 1 ? editor.filename : null"
/>
```

Reply:
Ok, I remember why. If I do not use `allow-filename`, I don't know when I can allow writing the filename or not. For example, inside `Editor.vue` in the `v-if="allowFilename"`, we need to know how much Editors are using to allow set the filename or not. With this `:filename="editors.length > 1 ? editor.filename : null"`, the filename always is allowed to set but, when just exists one Editor, the filename field is visible, but not visible in the render. Do you know what I mean?

Reply:
Maybe I can do something like `canSetFilename`